codemark/signup.py

```diff
@@ -0,0 +1,112 @@
+import getpass
+import codemark.firebase.database as FireDB
+from codemark.utils import print_info, print_error, print_warning, print_success
+from codemark.submit import generate_hash
+
+db = FireDB.FirebaseDB()
+
+def signup():
+    print_info("Are you a student or a professor? ((s)tudent/(p)rofessor")
+    resp = input('Response: ')
+    
+    # if the user is a student
+    if resp == 's':
```

Comment:
prefer using case insensitive checks for better UX, also match for complete word also

```py
if resp.lower() == "s" || resp.lower() == "student"
```